qadence/blocks/embedding.py

```diff
@@ -16,24 +14,38 @@
     parameters,
     uuid_to_expression,
 )
-from qadence.parameters import evaluate, stringify, torchify
+from qadence.parameters import evaluate, make_differentiable, stringify
+from qadence.types import ArrayLike, DifferentiableExpression, Engine, ParamDictType, TNumber
 
-StrTensorDict = dict[str, Tensor]
+
+def _concretize_parameter(engine: Engine) -> Callable:
+    if engine == Engine.JAX:
+        import jax.numpy as jnp
+
+        def concretize_parameter(value: TNumber, requires_grad: bool) -> ArrayLike:
+            fn, dtype = (jnp.array, jnp.float64) if requires_grad else (np.array, np.float64)
```

Comment:
I think jax automatically promotes inputs to jax, so just returning a numpy array *will not* do what you want it here...